crates/ark/src/lsp/completions/sources/composite/call.rs

```diff
@@ -67,6 +67,10 @@ pub(super) fn completions_from_call(
     // fn(name = value)
     //    ~~~~
     //
+
+    // JB: this caught my eye when doing the "no trailing parens" work
+    // it's a proper analysis of the completion site, e.g. investigating whether
+    // we know the site is CallNodePositionType::Value
```

Comment:
I found this other example of doing some formal analysis of the completion site to be interesting.